pkg/verifier/aws/entry_point.go

```diff
@@ -165,7 +165,7 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		"TIMEOUT":          vei.Timeout.String(),
 		"HTTP_PROXY":       vei.Proxy.HttpProxy,
 		"HTTPS_PROXY":      vei.Proxy.HttpsProxy,
-		"CACERT":           base64.StdEncoding.EncodeToString([]byte(vei.Proxy.Cacert)),
```

Comment:
We'll need to add a change to `legacy.Probe()` as it's still expecting a b64-encoded string

Reply:
Good catch here. Looking at the `userdata-template.yaml` for the legacy probe, I see this line:
```
echo "${CACERT}" | base64 --decode > /proxy.pem
```
What do you think about just changing that to
```
echo "${CACERT}" > /proxy.pem
```

Reply:
See [my above reply](https://github.com/openshift/osd-network-verifier/pull/262/files#r1704582179)